db/migrate/20170331194256_rename_users_recovery_code_to_personal_key.rb

```diff
@@ -0,0 +1,5 @@
+class RenameUsersRecoveryCodeToPersonalKey < ActiveRecord::Migration
+  def change
+    rename_column :users, :recovery_code, :personal_key
```

Comment:
I believe we do run migrations, but we have at least two app hosts, so one host will be running with code that looks for the old column name and so while the deploy is still happening (up to a few minutes) any requests that go to that host may fail.